pgxn/neon/file_cache.c

```diff
@@ -83,6 +84,9 @@ typedef struct FileCacheControl
 	uint64 generation; /* generation is needed to handle correct hash reenabling */
 	uint32 size; /* size of cache file in chunks */
 	uint32 used; /* number of used chunks */
+	uint32 limit; /* shared copy of lfc_size_limit */
+	uint64 hits;
+	uint64 misses;
```

Comment:
`lfc_cache_contains` is used only by prefetch. And It has no influence on performance, i.e. miss in this case doesn't mean that we need to download page - so there is actually no much difference between hit and miss. Prefetch logic depends on it, so it may be part of pgbench info. But I am also not sure that this information is so important for understanding prefetch activity.